thrift/src/main/java/com/linecorp/armeria/server/thrift/THttpService.java

```diff
@@ -105,9 +120,12 @@
      * <p>Currently, the only way to specify a serialization format is by using the HTTP session
      * protocol and setting the Content-Type header to the appropriate {@link SerializationFormat#mediaType()}.
      *
+     * @deprecated Use {@link THttpService#builder()}
+     *
      * @param implementation an implementation of {@code *.Iface} or {@code *.AsyncIface} service interface
      *                       generated by the Apache Thrift compiler
      */
+    @Deprecated
```

Comment:
Hi @trustin, with the new `THttpServiceBuilder` to build `THttpService`, we would be deprecating the static method here right? Is my understanding right? If yes, then I will make the changes everywhere these methods are used in code.

Reply:
I think some popular factory methods could be still provided as shortcuts. Let me leave some comments about which could be deprecated soon.